core/templates/domain/opportunity/exploration-opportunity-summary.model.ts

```diff
@@ -83,4 +87,23 @@ export class ExplorationOpportunitySummary {
     }
     return progressPercentage;
   }
+
+  getTranslationsInReviewCount(languageCode: string): number {
+    let inReviewCount = 0;
+    if (this.translationInReviewCount.hasOwnProperty(languageCode)) {
+      inReviewCount = (
+        (this.translationInReviewCount[languageCode]));
```

Comment:
You don't need the outer parens on this line.

Reply:
Done